src/workloads/execution/DensifyTimeseriesCollection.yml

```diff
@@ -0,0 +1,451 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of $densify stage with a numeric step
+  in timeseries collections.
+
+GlobalDefaults:
+  dbname: &db test
+  batchSize: &batchSize 30000
+  fieldName: &field "numeric"
+  unitName: &unit null
+  index: &index
+    keys: {numeric: 1}
+    options: {name: "numeric"}
+  smallStepSize: &smallStep {^RandomInt: {min: 3, max: 3}}
+  largeStepSize: &largeStep {^RandomInt: {min: 8, max: 8}}
```

Comment:
These numbers aren't ranges?

Reply:
The syntax is strange but it help encode smallStep's type. For instance, if I change it to: `smallStepSize: &smallStep 3` it crashes the server (`{ "ok" : 0.0, "errmsg" : "Upper bound, lower bound, and step must all have the same type", "code" : 5876900, "codeName" : "Location5876900" }`) so its ugly but the wrapper is necessary so the value 3 isn't read as a string, double, etc. This kind of syntax also already exists in other workloads in the repo

Reply:
Oh, so you wanted the step to always be 3 as opposed to 2-4? If so can you add a comment?

Reply:
done